src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -265,9 +270,12 @@
 
 (rf/reg-event-fx
  :wallet/set-collectible-to-send
- (fn [{db :db} [{:keys [collectible current-screen start-flow?]}]]
+ (fn [{db :db} [{:keys [collectible current-screen start-flow? entry-point]}]]
    (let [viewing-account?   (some? (-> db :wallet :current-viewing-account-address))
-         entry-point        (when-not viewing-account? :wallet-stack)
+         entry-point        (cond
```

Comment:
@ulisesmac, please correct me if I'm wrong but it looks like currently there are only 2 places where `set-collectible-to-send` called: when you select collectible in a send flow and from `send` button on collectible page itself. But this `cond` creates the perception that there are more cases. And I didn't find in what circumstances `entry-point` can get a value of `:wallet-stack` in current code.
So for the sake of clarity and better I'd fully remove this cond and replace later checks from
``` 
keep-tx-data? (#{:account-collectible-tab :wallet-stack} (-> db :wallet :ui :send :entry-point))
```
to something like
```
not-keep-tx-data? (= :account-send-button (-> db :wallet :ui :send :entry-point))
```

Wdyt?

Reply:
@vkjr 
First of all, thank you so much for paying attention to this piece of code.

Actually we have three entry points:
1. In the account page, when you click the send button- > pick an addess -> pick a collectible to send
2. in the account page, when you switch to the collectibles tab -> pick a collectible -> Send button
3. In the wallet home page, when you switch to the collectibles tab -> pick an owned collectible -> Send button.

So we need these three cases

Reply:
@ulisesmac, okay regarding three cases, but what about `keep-tx-data?`? It is always used with `not`, so why don't we simplify it?

Reply:
Yeah! Sorry for not replying to that, I'll change it! :)